tools/coverity-scan.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -euo pipefail
+
+# The following code will do:
+# 1. Download coverity scan package
+# 2. Run coverity scan when building rpms
+# 3. Upload coverity scan result to http://scan.coverity.com
+# Then we can login http://scan.coverity.com, find osbuild/image-builder, wait for analysis complete, and check the defect details
+echo "Downloading coverity scan package."
+sudo curl -o /tmp/cov-analysis-linux64.tgz https://scan.coverity.com/download/linux64 \
+        --form project="$COVERITY_SCAN_PROJECT_NAME" \
+        --form token="$COVERITY_SCAN_TOKEN"
```

Comment:
These variables need to be exported from the GitHub workflow file to this script using ${{ secrets.VAR_NAME }}

Reply:
Thanks, exported  these secrets in workflow file.